runtime/bref/0.2/serverless.yaml

```
@@ -0,0 +1,52 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: my-symfony-app
+configValidationMode: error
+useDotenv: true
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: prod
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
```

```suggestion
    stage: ${opt:stage,'prod'} 
    runtime: provided.al2
    environment:
        # Symfony environment variables
        APP_ENV: ${self:provider.stage}
```

By doing this, the user can easily deploy a prod/env Symfony app by specifying `--stage` when deploying.

At work, we used to work on multiples stages/Symfony environments:
- a "staging" environement: we automatically deploy branch `develop` or pull requests with `--stage dev`, which will run a Symfony app in `dev` environment 
- a "production" environment: we automatically deploy branch `main `with `--stage prod`, which will run our Symfony app in `prod` environment

What do you think?

---

This is not needed if you use the variable `${sls:stage}` which looks first for the cli option first, then the provider.stage value then defaults to dev if none is found.

---

+1 this is not needed, you should consider `stage` like a `defaultStage` configuration (we are considering renaming it to that in the future).

Also you might not want to mix Symfony env and stages, as you can have many stages (e.g. one per developer) which would not map to Symfony environments.

---

Definitely agree here, those are two different concepts that I would not mix to avoid confusion.

Also see my previous comment here about that https://github.com/symfony/recipes/pull/1001#discussion_r711788971

---

Definitely agree here, those are two different concepts that I would not mix to avoid confusion.

Also see my previous comment here about this https://github.com/symfony/recipes/pull/1001#discussion_r711788971

---

Alright, didn't know about that, thanks!